### PR TITLE
ADM-363 [frontend] [Metrics] Cycle time settings

### PR DESCRIPTION
**Pull request by yuyuFisher**
# Changes
1. store boardColumns
2. fix bug

### CODE REVIEW

File: frontend/__tests__/src/components/Metrics/MetricsStep/CycleTime.test.tsx [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
+import { setupStore } from '../../../utils/setupStoreUtil'
+
+let store = setupStore()
+
+beforeEach(() => {
+  store = setupStore()
+})
 
 const title = 'Cycle Time Settings'
 const defaultSelected = '----'
```
Comment (tingwang1234) [inline frontend/__tests__/src/components/Metrics/MetricsStep/CycleTime.test.tsx:14]: "Cycle Time Setting" or "Cycle Time Settings"

File: frontend/src/components/Metrics/MetricsStep/CycleTime/index.tsx [hunk 2]
```diff
@@ -3,27 +3,34 @@ import MetricsSettingTitle from '@src/components/Common/MetricsSettingTitle'
 import FlagCard from '@src/components/Metrics/MetricsStep/CycleTime/FlagCard'
 import { FormSelectPart } from '@src/components/Metrics/MetricsStep/CycleTime/FormSelectPart'
 import { ErrorDone } from '@src/components/Metrics/MetricsStep/CycleTime/style'
+import { useAppDispatch } from '@src/hooks/useAppDispatch'
+import { saveBoardColumns } from '@src/context/Metrics/metricsSlice'
 
 interface cycletimeProps {
```
Comment (fengxinhou) [inline frontend/src/components/Metrics/MetricsStep/CycleTime/index.tsx:9]: maybe cycleTimeProps?

**Review by tingwang1234 — APPROVED** [review]
LGTM

**Review by fengxinhou — APPROVED** [review]
LGTM

File: frontend/__tests__/src/components/Metrics/MetricsStep/CycleTime.test.tsx [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
 
-const title = 'Cycle Time Settings'
+let store = setupStore()
+
+beforeEach(() => {
+  store = setupStore()
+})
+
+const title = CYCLE_TIME_SETTINGS
 const defaultSelected = '----'
```
Comment (guzhongren) [inline frontend/__tests__/src/components/Metrics/MetricsStep/CycleTime.test.tsx:15]: the static variable should be `DEFAULT_SELECTED`
const title = CYCLE_TIME_SETTINGS is not necessary

File: frontend/cypress/pages/metrics.ts [hunk 4]
```diff
@@ -59,6 +60,10 @@ class Metrics {
     cy.contains('Next').click()
   }
 
+  checkCycleTime() {
```
Comment (guzhongren) [inline frontend/cypress/pages/metrics.ts:63]: Should not add assertion in POM,
ref to #博客大赛#Selenium之POM 设计模式

**Review by guzhongren — COMMENTED** [review]
Change with the comments

**Pull request merged**